### DISCUSSION TITLE
Any guidance on standing up via Nomad

### DISCUSSION THREAD

**Discussion by HardChalice** [netbox-community/netbox#6522] [Q&A] [answered]
I've been asked to look into using netbox via Nomad. I was just wondering if there was any documentation already available or not.

**Comment by ryanmerolle** [comment, marked as answer]
Nomad seems to be a virtualization orchestration platform sort of like docker-swarm, kubernetes, openshift, but wider supporting.

If it supports containers try leveraging https://github.com/netbox-community/netbox-docker
